static/bidder-info/adsyield.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "http://ads-pbs.open-adsyield.com/openrtb/{{.PublisherID}}?host={{.Host}}"
```

Comment:
```
~ » curl --location 'http://localhost:8000/openrtb2/auction' \                                                                                         oh0387@15852-p-mac
--header 'Content-Type: application/json' \
--data '{"id":"test-request-id","imp":[{"id":"test-imp-banner-id","banner":{"format":[{"w":300,"h":250},{"w":300,"h":300}]},"ext":{"prebid":{"bidder":{"adsyield":{"host":"audio.ortb.net","publisherId":123456}}}}}],"site":{"id":"1234","domain":"www.example.com","storeurl":"http://www.example.com","publisher":{"id":"1234","domain":"example.com"}}}'

{"id":"test-request-id","ext":{"errors":{"adsyield":[{"code":999,"message":"Post \"http://ads-pbs.open-adsyield.com/openrtb/123456?host=audio.ortb.net\": dial tcp: lookup ads-pbs.open-adsyield.com: no such host"}]},"warnings":{"general":[{"code":10002,"message":"debug turned off for account"}]},"responsetimemillis":{"adsyield":0},"prebid":{"auctiontimestamp":1683109393025}}}
```

@imedvedko,  received `no such host` error for sample request. could you confirm endpoint url.


```

Reply:
endpoint is now reachable 👍🏼